solidity/contracts/libraries/operator/GroupSelection.sol

```diff
@@ -404,4 +418,21 @@ library GroupSelection {
             delete self.candidate[self.tickets[i]];
         }
     }
+
+    function reactiveTicketSubmissionRounds(
+        uint256 possibleOperatorsCount,
+        uint256 groupSize
+    ) public view returns (uint256) {
```

Comment:
Should this one be public or `internal`?

Reply:
Not applicable now as we removed the dynamic submission timeout.